internal/cache/file/downloader/job_test.go

```diff
@@ -280,3 +309,90 @@ func (jt *jobTest) Test_updateFileInfoCache_Fail() {
 	lookupResult := jt.cache.LookUp(fileInfoKeyName)
 	ExpectTrue(lookupResult == nil)
 }
+
+// Note: We can't test Test_downloadObjectAsync_MoreThanSequentialReadSize as
+// the fake storage bucket/server in the testing environment doesn't support
+// reading ranges (start and limit in NewReader call)
+func (jt *jobTest) Test_downloadObjectAsync_LessThanSequentialReadSize() {
+	// Create new object in bucket and create new job for it.
+	objectName := "path/in/gcs/foo.txt"
+	objectSize := 50 * MiB
```

Comment:
ObjectSize = 50 MiB but content is "t". I think we can reduce the object Size further so that objectSize and content should match?

Reply:
Used random bytes instead of just 't' * size.